### DISCUSSION TITLE
Block duplicates from a site

### DISCUSSION THREAD

**Discussion by iliadaili** [proginosko/LeechBlockNG#370] [Ideas]
Hi! I'm sorry, this is the third suggestion in a row, but I'm more than satisfied with the extension and these are really just ideas for you to consider if you think that they're any good and that they may be worth the time and effort that would be required. The suggestion is the implementation of a feature that allows you to choose to block duplicates of a site that's already open, instead of blocking the site as a whole. That way, while the block was on, you'd only be allowed, for example, one open Twitter tab, or one open YouTube tab, etc. That could count for all windows, with you being able to literally have only one tab of that site open, or it could be a per window restriction, so you could have X tabs of that one site open in X windows. I think the former would make more sense as a Leechblock feature, but I'm also unaware of the limitations that the browsers impose on this idea I had. It's also easy to go from here to thinking of the possibility of choosing the number of tabs that would be allowed, but that could be going too far, considering the main idea itself isn't already a stretch.

**Comment by Kier60** [comment]
I also wanted to suggest this idea. the only way I can think of as a workaround rn is the delaying page which has the option "block only first accessed page when delaying page is used". thus, once the delaying timer is done, you can freely browse the blocked site on that tab. although it's not perfect and it's just a workaround; ideally I'd also like to be able to only allow 1 tab open to browse certain sites.
